### ISSUE TITLE
Sort results when retrieving orders

### ISSUE THREAD

**Issue by web4exposure-org** [justbetter/laravel-magento-client#33]
Is there a possibility to sort the results (ASC/DESC) when retrieving orders?
In the Magento docs, you need the `searchCriteria[sortOrders] `parameter, but I cannot find how to apply this.

**Comment by ramonrietdijk** [comment]
Hi @web4exposure-org,

It seems like we don't have a nice method for this yet. We will add this in the future. If you wish to make use of it sooner, feel free to make a pull request!

For the time being, you could append your own data to the `searchCriteria` like so:

```php
$searchCriteria = \JustBetter\MagentoClient\Query\SearchCriteria::make()
    ->where('state', 'processing')
    ->get();

$searchCriteria['searchCriteria[sortOrders][0][field]'] = 'created_at';
$searchCriteria['searchCriteria[sortOrders][0][direction]'] = 'desc';

foreach ($magento->lazy('orders', $searchCriteria) as $order) {
    // ...
}
```

**Comment by ramonrietdijk** [comment]
Hi @web4exposure-org,

The requested functionality has been released in version 2.3.0!

Now, you will be able to use it like so:

```php
$searchCriteria = \JustBetter\MagentoClient\Query\SearchCriteria::make()
    ->orderBy('entity_id')     // default ascending
    ->orderByDesc('entity_id') // multiple orders can be added
    ->get();
```
